orc8r/cloud/go/tools/swaggergen/main.go

```diff
@@ -100,6 +103,19 @@ func main() {
 		glog.Fatalf("Error parsing swagger spec dependency tree: %v\n", err)
 	}
 
+	if *generateStandAloneSpec && *targetFilepath != "swagger-common.yml" {
+		// Retrieve spec service name
+		splitPath := strings.Split(*targetFilepath, "/")
+		outFile := splitPath[len(splitPath)-4] + ".swagger.v1.yml"
```

Comment:
I think the basename of the filepath is already in the correct format -- we want same format, just different directory (partial vs. standalone)

Reply:
It seems to me that all the targetFilePaths are actually only "swagger.v1.yml" besides "swagger-common.yml". This seems to reflect what is passed into the gen.go files of each service.